_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -152,7 +152,8 @@ static mfxStatus SetRollingIntraRefresh(
     IntraRefreshState const & rirState,
     VADisplay    vaDisplay,
     VAContextID  vaContextEncode,
-    VABufferID & rirBuf_id)
+    VABufferID & rirBuf_id,
+    mfxU32 IntraRefreshBlockUnitSize = 0)
```

Comment:
can we add this information to ```IntraRefreshState```?

Reply:
I don't think so. IntraRefreshState is have to received from driver side according to capabilities.

Reply:
I think better to remove default value here. Also zero default contradicts to default of value 2, that you used below